genai-perf/genai_perf/logging.py

```diff
@@ -26,13 +26,43 @@
 
 import logging
 import logging.config
+import os
+from typing import Optional
 
-DEFAULT_LOG_FORMAT = "%(asctime)s [%(levelname)s] %(name)s:%(lineno)s - %(message)s"
-DEFAULT_DATE_FORMAT = "%Y-%m-%d %H:%M"
+from rich.highlighter import NullHighlighter
+from rich.logging import RichHandler
```

Comment:
This class be like

![gif](https://github.com/user-attachments/assets/7cd2db76-3953-46fe-85a9-e530d08e9ee8)